lib/app.rb

```diff
@@ -0,0 +1,83 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+require './format'
+require 'date'
```

Comment:
- [ ] Can you please croos check if you need to require 'date' here. I don't think you need it for your code to work. It throws the following error when I run your code
```
$ ruby lib/main.rb
Traceback (most recent call last):
        3: from lib/main.rb:1:in `<main>'
        2: from lib/main.rb:1:in `require_relative'
        1: from C:/Users/PC/Desktop/TSE/oop-training/lib/app.rb:6:in `<top (required)>'
C:/Users/PC/Desktop/TSE/oop-training/lib/app.rb:6:in `require_relative': cannot load such file -- C:/Users/PC/Desktop/TSE/oop-training/lib/date (LoadError)
```